compiler/rustc_target/src/spec/targets/x86_64_win7_windows_msvc.rs

```diff
@@ -1,16 +1,18 @@
-use crate::spec::{Target, base};
+use crate::spec::{SanitizerSet, Target, base};
 
 pub(crate) fn target() -> Target {
     let mut base = base::windows_msvc::opts();
+    base.vendor = "win7".into();
     base.cpu = "x86-64".into();
+    base.features = "+cx16,+sse3,+sahf".into();
     base.plt_by_default = false;
-    base.max_atomic_width = Some(64);
-    base.vendor = "win7".into();
+    base.max_atomic_width = Some(128);
```

Comment:
`"+cx16,+sse3,+sahf"` and 128-bit atomic are requirements of Windows 8.1+ 64-bit, and may not available on Windows 7.

Reply:
Removed. Thanks.